scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -167,6 +168,68 @@ impl ReplicatorConsumer {
         Ok(())
     }
 
+    // Recreates INSERT/DELETE/UPDATE statement on non-frozen list.
+    async fn update_list<'a>(
+        &self,
+        column_name: &str,
+        data: &'a CDCRow<'_>,
+        values_for_update: &mut Vec<Option<&'a CqlValue>>,
+        values_for_delete: &[&CqlValue],
```

Comment:
Hmmm, why is this called `values_for_delete`? From the code it seems that the values passed here are primary key values. And that's how they are used too - for the placeholders in the key condition.

Is the variable name as such because it is used in a `DELETE` statement?
It's quite confusing, the parameter names should be chosen so that the caller of the function can understand them - they are part of the interface. They shouldn't be named based on the internal usage by the implementation of that function, IMO. How does the caller know to put primary key into `values_for_delete`?